autoload/neomake/makers/ft/scss.vim

```diff
@@ -1,12 +1,22 @@
 " vim: ts=4 sw=4 et
 
 function! neomake#makers#ft#scss#EnabledMakers()
-    return ['scsslint']
+    return ['sasslint', 'scsslint']
 endfunction
 
 function! neomake#makers#ft#scss#scsslint()
     return {
-        \ 'exe': 'scss-lint',
-        \ 'errorformat': '%f:%l [%t] %m'
-    \ }
+        \   'exe': 'scss-lint',
+        \   'errorformat': '%f:%l [%t] %m'
+        \ }
 endfunction
+
+function! neomake#makers#ft#scss#sasslint()
+    return {
+        \   'exe':  'sass-lint',
+        \   'args': [ '--no-exit', '--verbose', '--format=compact' ],
```

Comment:
When editing the PR, please remove the extra space around the outer list items here.